emscripten.py

```diff
@@ -631,10 +630,6 @@ def add_standard_wasm_imports(send_items_map):
 
   if settings.RELOCATABLE:
     send_items_map['__indirect_function_table'] = 'wasmTable'
-    if settings.WASM_EXCEPTIONS:
-      send_items_map['__cpp_exception'] = '___cpp_exception'
-    if settings.SUPPORT_LONGJMP == 'wasm':
-      send_items_map['__c_longjmp'] = '___c_longjmp'
```

Comment:
These tags used to not show up in the `import` .. but now that do so we can rely on the normal symbol resolution to find them in the JS library.